packages/docs/concepts/connections-and-requests.yaml

```diff
@@ -43,12 +37,16 @@ _ref:
         type: Markdown
         properties:
           content: |
+            To implement requests, the following steps are required:
+            - define a connection.
+            - define a request.
+            - call the request using a [request action](/Request).
+            - use returned data by making use of the [request operator](/_request).
```

Comment:
I think thes should be `[Request](/Request) action` and `[_request](/_request)operator`.